Gemfile

```diff
@@ -42,13 +42,17 @@ gem 'tzinfo-data', platforms: %i[mingw mswin x64_mingw jruby]
 # Reduces boot times through caching; required in config/boot.rb
```

Comment:
README should have steps to set the project up locally and run the tests. This way it looks more professional.